### PR TITLE
feat: add explore layout with fake data

### PR DESCRIPTION
**Pull request by RaphaelEscrig**
This PR aims to implement the _explore_ page of the portal.
Please have a look at this issue for more details: https://github.com/okp4/portal-web/issues/26

URL to test: http://localhost:3000/explore

The build is not working because it needs the ui with the last List / ListItem version. In order to test this PR, you need to manually add the good version of the lib.

👉 As it is not possible to disable SSR for the whole page, we had to split the page into sub-components in order to disable SSR each time a document was used.

### CODE REVIEW

File: src/components/explore/exploreList/ExploreList.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  }, [range, sortBy])
+
+  const onListItemClick = useCallback(
+    (item: DeepReadonly<ExploreItem>) => (): void => {
+      if (item.type === 'dataspace') {
+        router.push(`/dataspace/${item.dataspaceId}`)
+        return
+      }
+
+      router.push(`/dataspace/${item.dataspaceId}/${item.type}/${item.id}`)
```
Comment (JoeyDjr) [inline src/components/explore/exploreList/ExploreList.tsx:102]: You can remove this unnecessary `return`
```suggestion
      if (item.type === 'dataspace') {
        router.push(`/dataspace/${item.dataspaceId}`)
      } else {
        router.push(`/dataspace/${item.dataspaceId}/${item.type}/${item.id}`)
      }
```

**Review by JoeyDjr — CHANGES_REQUESTED** [review]
LGTM, juste a remark

File: src/pages/api/fake/explore/index.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+const RANGE_DEFAULT_VALUE = 10
+
+const getRange = (range: string | undefined): number => {
+  if (range === undefined) return RANGE_DEFAULT_VALUE
+
+  return parseInt(range)
+}
+
+// eslint-disable-next-line @typescript-eslint/prefer-readonly-parameter-types
+const handler = (req: NextApiRequest, res: NextApiResponse): void => {
```
Comment (ErikssonJoakim) [inline src/pages/api/fake/explore/index.ts:13]: No need to disable the linter here, parameters can be readonly

File: src/components/pageTitle/pageTitle.scss [hunk 3]
```diff
@@ -0,0 +1,7 @@
+.okp4-page-title {
+  padding-left: 5vw;
+
+  @media screen and (max-width: 1300px) {
```
Comment (ErikssonJoakim) [inline src/components/pageTitle/pageTitle.scss:4]: Imo the title should be centered when filter moves above the list. Same for the list configuration.

It looks weird to have the config and title not centered when the list is.

**Review by ErikssonJoakim — CHANGES_REQUESTED** [review]
Got some last minute remarks, otherwise it looks good to me :)

**Review by JoeyDjr — APPROVED** [review]

**Comment by fredericvilcot** [comment]
@RaphaelEscrig 
We can't review this PR as the Ci fails.
Thanks

**Comment by RaphaelEscrig** [comment]
>

**Pull request closed without merging**